packages/pxweb2-ui/src/index.ts

```diff
@@ -1,30 +1,33 @@
+export * from './../style-dictionary/dist/js/css-variables';
+export * from './../style-dictionary/dist/js/fixed-variables';
 export * from './lib/components/Alert/Alert';
+export * from './lib/components/BottomSheet/BottomSheet';
 export * from './lib/components/Button/Button';
 export * from './lib/components/Checkbox/Checkbox';
 export * from './lib/components/Contact/Contact';
+export * from './lib/components/Contact/Contact';
 export * from './lib/components/EmptyState/EmptyState';
 export * from './lib/components/Icon/Icon';
 export * from './lib/components/Link/Link';
+export * from './lib/components/Notes/MandatoryNotes';
+export * from './lib/components/Notes/MandatoryTableNotes';
+export * from './lib/components/Notes/MandatoryVariableNotes';
+export * from './lib/components/Notes/NonMandatoryNotes';
+export * from './lib/components/Notes/NoNotes';
+export * from './lib/components/Notes/noteCollection';
+export * from './lib/components/Notes/SymbolExplanationNotes';
 export * from './lib/components/Search/Search';
 export * from './lib/components/Select/Select';
+export * from './lib/components/Spinner/Spinner';
```

Comment:
Spinner was the missing export - all the others have been reordered alphabetically only, to increase redability.